Minesweeper.java

```diff
@@ -109,31 +166,32 @@ private void computeNeighboringBombs(Cell cell) {
                         continue;
                     }
                     // skip out of bounds cases
-                    if(y < 0 || x < 0 || y >= this.gridSize || x >= this.gridSize) {
+                    if(y < 0 || x < 0 || y >= Game.gridSize || x >= Game.gridSize) {
                         continue;
                     }
 
-                    if (this.cells[y][x].isBomb) {
+                    if (Game.cells[y][x].isBomb) {
                         neighboringBombs++;
                     }
                 }
             }
-
             cell.setNeighboringBombs(neighboringBombs);
+            return neighboringBombs;
         }
 
         public Game(int gridSize, int bombAmount){
             this.cellSize = 35;
-            this.gridSize = gridSize;
+            Game.gridSize = gridSize;
             this.bombAmount = bombAmount;
-            this.cells = new Cell[this.gridSize][this.gridSize];
+            Game.cells = new Cell[Game.gridSize][Game.gridSize];
 
-            setSize(this.gridSize * this.cellSize, this.gridSize * this.cellSize);
-            setLayout(new GridLayout(this.gridSize, this.gridSize));
+            setSize(Game.gridSize * this.cellSize, Game.gridSize * this.cellSize);
```

Comment:
why are we using a mix of `this` and `Game` here?
```suggestion
            setSize(this.gridSize * this.cellSize, this.gridSize * this.cellSize);
```